cve/system/hsp-cve-2019-13139-docker-build.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: security.kubearmor.com/v1
+kind: KubeArmorHostPolicy
+metadata:
+  name: hsp-cve-2019-13139-docker-build
+spec:
+  tags: ["CVE-2019-13139", "Docker-ce", "V-18.03.1", "git RCE"]
+  message: "Docker Build Command is executed. Check Audit Logs"
+  nodeSelector:
+    matchLabels:
+      kubernetes.io/arch: amd64
+      kubernetes.io/hostname: gke-cluster-1-default-pool-37b9f014-0vth
```

Comment:
The hostname are not generic to all cluster, can we keep it generic if it's necessity or add comment